scripts/kubernetes/serversChart/templates/webuiDeployment.yaml

```diff
@@ -0,0 +1,43 @@
+#/*********************************************************************
+# Copyright (c) Intel Corporation 2020
+# SPDX-License-Identifier: Apache-2.0
+#**********************************************************************/
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: azure-webui
+spec:
+  replicas: {{ .Values.ui.replicaCount }}
+  selector:
+    matchLabels:
+      app: azure-webui
+  template:
+    metadata:
+      labels:
+        app: azure-webui
+    spec:
+      imagePullSecrets:
+        - name: {{ .Values.images.pullSecret }}
+      containers:
+      - name: azure-webui
+        image: {{ .Values.images.ui }}
+        env:
+          - name: "NODE_ENV"
+            value: "production"
+          - name: "MPS_SERVER"
+            value: "https://{{ .Values.mps_web.staticIPAddress }}:{{ .Values.mps_web.webPort }}"
```

Comment:
why are we using a static ip addresses instead of container/dns name?

Reply:
there are certs/cores issues with changing them. if you like i can create a defect to look into it.